test/parallel/test-internal-modules-expose.js

```diff
@@ -5,3 +5,4 @@ require('../common');
 const assert = require('assert');
 
 assert.strictEqual(typeof require('internal/freelist').FreeList, 'function');
+assert(!('_exposeInternals' in process), 'no process property is leaked');
```

Comment:
see https://github.com/nodejs/node/blob/84602845c6f2a015b7b214751543d0793e45a5a9/src/node.cc#L1219-L1221, this is already common practice, just usually from C++.